server/src/main/java/org/apache/uniffle/server/DefaultFlushEventHandler.java

```diff
@@ -126,6 +126,11 @@ private void handleEventAndUpdateMetrics(ShuffleDataFlushEvent event, Storage st
       if (e instanceof EventInvalidException) {
         return;
```

Comment:
Why wouldn't you release an invalid event?

Reply:
Reasonable. Maybe it is worth mentioning this fact here. Something like
```suggestion
        // invalid events have already been released / cleaned up so no need to call event.doCleanup() here
        return;
```

Reply:
@EnricoMi Your commit is merged.

Reply:
> This is because once the ShuffleBuffer associated with the appId expires, its corresponding bufferPool will be cleared by a scheduled task. Consequently, the event related to the appId will become inValid, and there is no need to repeatedly release memory, as the scheduled task has already taken care of releasing the buffer memory for all events associated with that appId.

Absolutely right.